index.js

```diff
@@ -1,27 +1,27 @@
+/*
+This is a proposal on how to achieve a safety net for modules that are removed from the app.
+If that is the case we want to know at start time that something is not correct.
 
-// // production code
-// const doMagic = require('./command/do-magic').doMagic
-// doMagic({color: 'magenta', message: 'You are one nice cow!'})
-//
-//
-//
-// // test code
-// const createDoMagicCommand = require('./command/do-magic').createDoMagicCommand
-//
-// const testDoMagic = createDoMagicCommand({
-//   chalk: require('chalk'),
-//   cowsay: {
-//     say({text}) {
-//       return `🐮  ${text}`
-//     }
-//   },
-//   log: console.log
-// })
-//
-// testDoMagic({color: 'blue', message: 'Test cows everywhere!'})
+It's easly achiavable with direct require of those modules and avoiding index.js:
+ */
 
 
-// throws an erro
-const doIndexMagic = require('./command')
+// production code
+const doMagic = require('./command/do-magic').doMagic
+doMagic({color: 'magenta', message: 'You are one nice cow!'})
```

Comment:
@Restuta There is still the problem that if somebody exports the `doMagic` function directly, or makes a typo, etc, this line will fail at runtime instead of at startup time.  I feel like nesting the commands under a folder, while more boilerplate, is the most fool-proof way around the problem (and provides the quickest feedback for a dev that they did something wrong).  Something like:

```js
// someCommand/index.js
const createCommand = require('./create-command')

module.exports = createCommand({ dependencies, provided })

// create-command
module.exports = ({ dependencies, provided }) => P.coroutine(function* someCommand() {
  ...
})
```